yb-voyager/src/cp/yugabyted/yugabyted.go

```diff
@@ -0,0 +1,593 @@
+		}
+
+	}
+	cp.createAndSendUpdateRowCountEvent(updateExportedRowCountEvents)
+}
+
+func (cp *YugabyteD) ImportSchemaStarted(importSchemaEvent *controlPlane.ImportSchemaStartedEvent) {
+	cp.createAndSendEvent(&importSchemaEvent.BaseEvent, "IN PROGRESS", "")
+}
+
+func (cp *YugabyteD) ImportSchemaCompleted(importSchemaEvent *controlPlane.ImportSchemaCompletedEvent) {
+	cp.createAndSendEvent(&importSchemaEvent.BaseEvent, "COMPLETED", "")
+}
+
+func (cp *YugabyteD) SnapshotImportStarted(snapshotImportEvent *controlPlane.SnapshotImportStartedEvent) {
+	cp.createAndSendEvent(&snapshotImportEvent.BaseEvent, "IN PROGRESS", "")
+}
+
+func (cp *YugabyteD) SnapshotImportCompleted(snapshotImportEvent *controlPlane.SnapshotImportCompletedEvent) {
+	cp.createAndSendEvent(&snapshotImportEvent.BaseEvent, "COMPLETED", "")
+}
+
+func (cp *YugabyteD) UpdateImportedRowCount(
+	snapshotImportTableMetrics []*controlPlane.UpdateImportedRowCountEvent) {
+
+	cp.Mutex.Lock()
+	defer cp.Mutex.Unlock()
+
+	var updateImportedRowCountEvents []*controlPlane.BaseUpdateRowCountEvent
+	for _, updateImportedRowCountEvent := range snapshotImportTableMetrics {
+
+		lastRowCountUpdateTime, check := cp.lastRowCountUpdate[updateImportedRowCountEvent.TableName]
+
+		if updateImportedRowCountEvent.Status == "COMPLETED" {
+			updateImportedRowCountEvents = append(updateImportedRowCountEvents, &updateImportedRowCountEvent.BaseUpdateRowCountEvent)
+		} else if !check {
+			cp.lastRowCountUpdate[updateImportedRowCountEvent.TableName] = time.Now()
+			updateImportedRowCountEvents = append(updateImportedRowCountEvents, &updateImportedRowCountEvent.BaseUpdateRowCountEvent)
+		} else if !lastRowCountUpdateTime.Add(time.Second * 5).After(time.Now()) {
+			cp.lastRowCountUpdate[updateImportedRowCountEvent.TableName] = time.Now()
+			updateImportedRowCountEvents = append(updateImportedRowCountEvents, &updateImportedRowCountEvent.BaseUpdateRowCountEvent)
+		}
+
+	}
+	cp.createAndSendUpdateRowCountEvent(updateImportedRowCountEvents)
+}
+
+func (cp *YugabyteD) MigrationEnded(migrationEndedEvent *controlPlane.MigrationEndedEvent) {
+}
+
+func (cp *YugabyteD) panicHandler() {
+	if r := recover(); r != nil {
+		// Handle the panic for eventPublishers
+		log.Errorf(fmt.Sprintf("Panic occurred:%v\n"+
+			"Stack trace of panic location:\n%s", r, string(debug.Stack())))
+	}
+}
+
+func (cp *YugabyteD) getConnPool() (*pgxpool.Pool, error) {
+	var err error
+	err = nil
+	if cp.connPool == nil {
+		log.Warnf("No Connections to YugabyteD DB found. Creating a connection pool...")
+		err = cp.connect()
+	}
+
+	return cp.connPool, err
+}
+
+func (cp *YugabyteD) reconnect() error {
+	var err error
+	cp.disconnect()
+	for attempt := 1; attempt < 5; attempt++ {
+		err = cp.connect()
+		if err == nil {
+			return nil
+		}
+		log.Infof("Failed to reconnect to the target database: %s", err)
+		time.Sleep(time.Second)
+	}
+	return fmt.Errorf("reconnect to target db: %w", err)
```

Comment:
Nit: do not refer it as "target db" -- always refer it as `yugabyted db`.

Reply:
Done. Replaced all instances of `target db` with `yugabyted db`.